### PR TITLE
Fixes for Windows test.

### PR DESCRIPTION
**Pull request by marsupial**
Whether these functions aren't being exported because of CMake, the build configuration, or MSVC thinking they shouldn't be is unknown to me.

### CODE REVIEW

File: test/Driver/E.C [hunk 1]
```diff
@@ -6,10 +6,9 @@
 // LICENSE.TXT for details.
 //------------------------------------------------------------------------------
 
-// RUN: %cling -dM -E -xc++ /dev/null | grep CLING | FileCheck %s
+// RUN: echo "" | %cling -dM -E -xc++ -std=c++11 | grep CLING | FileCheck %s
```
Comment (karies) [inline test/Driver/E.C]: Why do we need to pass `-std=c++11`?

File: test/Driver/E.C [hunk 2]
```diff
@@ -6,10 +6,9 @@
...
 
 // CHECK: #define __CLING__ 1
 // CHECK: #define __CLING__CXX11 1
-// CHECK: #define __CLING__{{GNUC|clang}}
```
Comment (karies) [inline test/Driver/E.C]: Can we instead add a `__CLING__MSVC`?

File: test/Driver/E.C (hunk 1 above)
Comment (marsupial) [inline test/Driver/E.C]: I want to force c++11 for the __CLING__CXX11 check.
At some point (MSVC) c++14/17 may be the default?

File: test/Driver/E.C (hunk 2 above)
Comment (marsupial) [inline test/Driver/E.C]: Updated.
Seems a wiser solution.

**Comment by karies** [comment]
Yippee, thanks! :-) Merged.

**Pull request closed without merging**